### ISSUE TITLE
A few (new?) small/rare issues with green score comparison bar

### ISSUE THREAD

**Issue by tchesket** [Splamy/ScoreSaberEnhanced#17]
I have found 3 small strange issues. 

1. #### There is occasionally a small green bar that shows up when none should, ie when comparing someone's scores with their own scores from the drop-down, the scores will be identical but it shows up as if one is slightly higher. Only seen it happen on scores that are very low in pp. seems to happen when sorting by recent and by top, can't find a definite, consistent pattern as to when/where it happens. screens attached:

   ![phantom_green_bar](https://user-images.githubusercontent.com/40137713/87356451-b50ac600-c527-11ea-82f2-5ef4a8610e2f.png)
  ![phantom_green_bar_2](https://user-images.githubusercontent.com/40137713/87356577-e8e5eb80-c527-11ea-9391-be253d038f44.png)

2. #### This one is odd, but for some reason starting from page 265, when sorting by recent scores (but not by top scores), every score except the first one on page 265 and all thereafter are entirely filled up with green, even though the scores being compared are the same exact score by the same user. I'm not sure if this happens for all users or not, I only have observed this on one user's page (most people do not have 300+ pages of scores). screenshot:
    (this one sometimes doesn't happen, it usually does, however I can't figure why that is)
  ![page_265_weird_issue](https://user-images.githubusercontent.com/40137713/87357713-1df33d80-c52a-11ea-82a3-ad18f09f353e.png)

3. #### This might be an issue with scoresaber itself, not sure, but I've found two scores listed that appear to be 0, ie they aren't real scores, and the comparison tool shows the text \"\<No Data>" where the selected user's accuracy should be, final screenshot (you can also see the previous issue is still present here, on page 320 the final page of this user's scores: 
   ![last_page_no_data](https://user-images.githubusercontent.com/40137713/87358292-1aac8180-c52b-11ea-9a58-816dad9ccd5f.png)

**Comment by Splamy** [comment]
1. Might be due to the fact that the result from the scoresaber api is more precise than the displayed accuracy on the site, since for the visual bar the exact value is taken from the cache-db

2. Yeah scoresaber after some time switches from 'score' to 'accuracy' for songs. I don't know when or why that happens, but this is also a change which SSE by the nature of the lazy updating won't catch unless you do a redownload of the whole user cache (for that user).

3. Thats weird, 'no data' would mean that at the point of caching the song data, none of the possible scoring options ("pp", "acc" or "score") was available. Do you have this cache for a long time already? It might be before the usage of the new scoresaber api. Otherwise that's hard to explain or debug...

**Comment by tchesket** [comment]
Nope, I deleted said user after I updated and re-cached the scores. That's the first I've seen of that either. You can see in the screenshot, their score according to scoresaber is "0.00% accuracy" (which is weird in itself), and the script shows no data. Yeah that one is beyond me, no clue what to make of it.

**Comment by Splamy** [comment]
Ah I see, I have a sanity check for `song_a.accuracy > 0` because I assumed that shouldn't be possible xD, yeah should be an easy visual fix

**Comment by tchesket** [comment]
Yeah they must have just put no fail on and missed every note in the song, I'm surprised ss accepts those lol. Okay well looks like everything is more or less explained then.
